node/src/effect/announcements.rs

```diff
@@ -204,27 +204,52 @@ where
     }
 }
 
-/// A BlockExecutor announcement.
+/// A ContractRuntime announcement.
 #[derive(Debug)]
-pub enum BlockExecutorAnnouncement {
+pub enum ContractRuntimeAnnouncement {
     /// A new block from the linear chain was produced.
-    LinearChainBlock {
-        /// The block.
+    LinearChainBlock(Box<LinearChainBlock>),
```

Comment:
Maybe it's a matter of naming but _contract runtime_ producing _linear chain blocks_ doesn't sound good. Maybe we should rename `ContractRuntime` now. Different PR though.